### ISSUE TITLE
Additional DPTs?

### ISSUE THREAD

**Issue by greiginsydney** [mhthies/knxdclient#6]
Hi Michael,

Thanks for the [Merge](https://github.com/mhthies/knxdclient/pull/5), which kicked my year off to a lovely start.

I was wondering what your appetite is for some more DPTs, or if you prefer I should fork instead?

These look relatively easy to add:

- Scene Info 26 is very similar to 18 ("SCENE_INFO"?)
- Energy 29 is the same construct as 2 x 13's, so presumably "INT64"
- Scene config 238 is similar to 26 ("SCENE_CONTROL")
- HVAC 212 & 213 are just combo's of 16 bit signed int's
- HVAC 23 is just a 2-bit version of ENUM DPT 20

I was going to do them one at a time. Hopefully each will be quicker than the first time.

Let me know what you think.

\- Greig.

**Comment by mhthies** [comment]
Yes, of course, feel free to create Pull Requests to add them to this package! :)

If you need one of them at short notice, just let me know and I'll create a new release.

**Comment by greiginsydney** [comment]
Thanks @mhthies. I don't have any immediate need, but I thought I'd try to add a few easier ones.

I'll do a few one at a time and send them through.

**Comment by greiginsydney** [comment]
Hey @mhthies, I think I've created all the new DPTs that I can envisage a need for at the moment. Could I trouble you to publish a release at your next convenience?

Thanks.

\- Greig.

**Comment by mhthies** [comment]
Done: https://github.com/mhthies/knxdclient/releases/tag/v1.2.0 / https://pypi.org/project/knxdclient/1.2.0/ :)

**Comment by greiginsydney** [comment]
Love your work! Thanks @mhthies
